core/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm, UsernameField
+from .models import User
+
+class CustomUserCreationForm(UserCreationForm):
```

Comment:
Jeżeli chcesz zmodyfikować UserCreationForm, to nie deklaruj wszystkiego od nowa, Ten formularz działa jak ModelForm (bo po nim dziedziczy), wystarczy, że w class Meta wrzucisz wymagane pola w "fields", labelki w "labels", widgety w "widgets" jako atrybuty class Meta.